### PR TITLE
Renamed `Strings.swift` to `Constants.swift`

### LINKED ISSUE #498: Rename Strings to Constants
Either rename `String` to `Constants` and store both links and localized string etc. Or create a new file called `Constants` to store the urls.

See comment: https://github.com/rnystrom/GitHawk/pull/494#discussion_r143076287

### PR DESCRIPTION
**Pull request by weyert**
Renamed `Strings.swift` to `Constants.swift`, fixes #498
Added the product urls to `Constants.swift`, partly fixes #494

### CODE REVIEW

**Comment by rnystrom** [comment]
To be consistent, can we move all of the strings into a `Strings` nested enum?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Sherlouk** [comment]
@rnystrom Also why an enum? Can these not be `struct`s?

**Comment by weyert** [comment]
I am happy with both. Please advise the change you would like to have :+

**Comment by BasThomas** [comment]
Because using an enum makes it so you can’t initiate it, a struct could be initiated (and you don’t want that) @Sherlouk 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Could conflicts @weyert if you don't mind!

File: Classes/Views/Constants.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        static let unread = NSLocalizedString("Unread", comment: "")
+        static let ok = NSLocalizedString("OK", comment: "")
+        static let cancel = NSLocalizedString("Cancel", comment: "")
+        static let yes = NSLocalizedString("Yes", comment: "")
+        static let no = NSLocalizedString("No", comment: "")
+        static let signin = NSLocalizedString("Sign In", comment: "")
+        static let signout = NSLocalizedString("Sign out", comment: "")
+        static let open = NSLocalizedString("Open", comment: "")
+        static let close = NSLocalizedString("Closed", comment: "")
+        static let closed = NSLocalizedString("Closed", comment: "")
```
Comment (Sherlouk) [inline Classes/Views/Constants.swift]: 📛 Something went wrong here! `close` should be `Close` not `Closed` as we have duplication!

**Comment by weyert** [comment]
Think it's all good now, Xcode keeps selecting my iPad so it keeps changing the `DEVELOPMENT_TEAM` I have patched your values back.

**Pull request merged**